src/env/as_env.rs

```diff
@@ -130,3 +130,32 @@ pub fn assembly_script_date_now(mut ctx: FunctionEnvMut<ASEnv>) -> ABIResult<f64
     let ret = utime as f64;
     Ok(ret)
 }
+
+#[named]
+pub fn assembly_script_console_log(
+    mut ctx: FunctionEnvMut<ASEnv>,
+    message: StringPtr,
+) -> ABIResult<()> {
+    let env = ctx.data().clone();
+    if cfg!(not(feature = "gas_calibration")) {
+        sub_remaining_gas_abi(&env, &mut ctx, function_name!())?;
+    }
+
+    let memory = ctx
+        .data()
+        .get_wasm_env()
+        .memory
+        .as_ref()
+        .expect("Failed to get memory on env")
+        .clone();
+    let message_ = message
+        .read(&memory, &ctx)
+        .map_err(|e| wasmer::RuntimeError::new(e.to_string()));
```

Comment:
Isn't possible to use `?` here ? If not, above you could if the destructing pattern `if let Ok(message) = message_ {...} else {...}`

Reply:
Fixed